### PR TITLE
Removed domain and range from gist:isGovernedBy

### LINKED ISSUE #899: Remove domain and range from gist:isGovernedBy (was "Expand domain (and range?) of `governs`")
Proposal to expand the domain of `governs` to include `Commitment`.

Would make sense to add `Commitment` to the range as well.

### PR DESCRIPTION
**Pull request by turingtest37**
Fixes #899
Removed domain and range from gist:isGovernedBy

### CODE REVIEW

**Review by uscholdm — APPROVED** [review]
Looks good to me.

**Review by brett-heffelfinger — APPROVED** [review]
Looks good to me.

File: docs/release_notes/Issue-899-isGovernedBy.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Major Updates
+
+- Removed domain and range from `gist:isGovernedBy`
+  Issue [899] (https://github.com/semanticarts/gist/issues/899)
```
Comment (rjyounes) [inline docs/release_notes/Issue-899-isGovernedBy.md:4]:
```suggestion
- Removed domain and range from `gist:isGovernedBy`. Issue [899](https://github.com/semanticarts/gist/issues/899).
```

**Review by rjyounes — CHANGES_REQUESTED** [review]
OWL good, one correction on release note.

**Pull request merged**